src/sources/apache_metrics/parser.rs

```diff
@@ -0,0 +1,919 @@
+use crate::event::metric::{Metric, MetricKind, MetricValue};
+use chrono::{DateTime, Utc};
+use lazy_static::lazy_static;
+use std::collections::BTreeMap;
+use std::collections::HashMap;
+use std::error::Error;
+use std::fmt;
+
+lazy_static! {
+    static ref SCOREBOARD: HashMap<char, &'static str> = vec![
+        ('_', "waiting"),
+        ('S', "starting"),
+        ('R', "reading"),
+        ('W', "sending"),
+        ('K', "keepalive"),
+        ('D', "dnslookup"),
+        ('C', "closing"),
+        ('L', "logging"),
+        ('G', "finishing"),
+        ('I', "idle_cleanup"),
+        ('.', "open"),
+    ]
+    .into_iter()
+    .collect();
+}
+
+/// enum of mod_status fields we care about
+enum StatusFieldStatistic<'a> {
+    ServerUptimeSeconds(f64),
+    TotalAccesses(f64),
+    TotalKBytes(f64),
```

Comment:
I thought about that, but figured since they are just passed through directly to the `Metric` type which wants a `f64`s anyway, that converting back and forth didn't really give as much. However, I can see the argument for modeling the data as it is. I'll make that change.